prow/config/config.yaml

```
@@ -80,6 +80,81 @@ postsubmits:
         - image: alpine
           command:
             - /bin/printenv
+
+  nephio-project/free5gc-operator:
+  - name: build-push-image-free5gc-operator-release-conf
+    cluster: default
+    always_run: true
+    branches:
+    - "^v.*$"
+    annotations:
+      description: Build and Push Image to DockerHub when release tag is created
+    decorate: true
+    decoration_config:
+      censor_secrets: true
+    max_concurrency: 1
```

Do we have to [restrict the number](https://github.com/kubernetes/test-infra/blob/master/prow/apis/prowjobs/v1/types.go#L174-L179) of jobs to run in parallel?
```suggestion
    max_concurrency: 0
```